spikeextractors/extractors/bindatrecordingextractor/bindatrecordingextractor.py

```diff
@@ -17,18 +17,21 @@ class BinDatRecordingExtractor(RecordingExtractor):
         {'name': 'dtype', 'type': 'np.dtype', 'title': "The dtype of underlying data"},
         {'name': 'recording_channels', 'type': 'list', 'value':None, 'default':None, 'title': "List of recording channels"},
         {'name': 'frames_first', 'type': 'bool', 'value':True, 'default':True, 'title': "Frames first"},
-        {'name': 'offset', 'type': 'int', 'value':0, 'default':0, 'title': "Offset in binary file"},
-        {'name': 'probe_path', 'type': 'str', 'title': "Path to probe file (csv or prb)"},
+        {'name': 'geom', 'type': 'list', 'title': "list or array with channel locations"},
```

Comment:
Why this here ?

Reply:
this what? geom? You could pass an array with the locations

Reply:
Why this in this PR I mean.

Reply:
geom

Reply:
it was already there. The gui params was wrong

Reply:
I did not include geom in the GUI params on purpose. It figure it is easier to specify which extractors have default locations (not binary) and for the ones that don't, I force them to give a probe file (for now).

Reply:
So lets remove this 'goem' in gui_params for now. No ?
And then we can merge.

Reply:
Yes sounds good.

Reply:
done